### PR TITLE
Use dedicated PHPUnit assertions for better error messages

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: test/classes/phing/components/TargetTest.php [hunk 1]
```diff
@@ -51,8 +51,8 @@ public function testHiddenTargets()
         exec($cmd, $out);
         $out = implode("\n", $out);
         $offset = strpos($out, 'Subtargets:');
```
Comment (siad007) [inline test/classes/phing/components/TargetTest.php]: It seems to me that this line could be removed then.

Comment (carusogabriel) [reply test/classes/phing/components/TargetTest.php]: Gonna fix it and the other tests that broked.

Comment (carusogabriel) [reply test/classes/phing/components/TargetTest.php]: @siad007 How do I run tests locally?

I didn't found a `CONTRIBUING.MD`, as well. running `./.travis.sh` gave me:
```bash
carusogabriel@caruso:~/Code/phing$ sudo ./.travis.sh 
[sudo] password for carusogabriel: 
config-set succeeded

Auto-discover pear channels and upgrade ...
config-set succeeded
Updating channel "pear.php.net"
Channel "pear.php.net" is up to date
Channel "pear.phing.info" is already initialized
carusogabriel@caruso:~/Code/phing$
```

Comment (mrook) [reply test/classes/phing/components/TargetTest.php]: @carusogabriel `.travis.sh` is only used/usable within the Travis CI context. https://github.com/phingofficial/phing/blob/master/README.md details how to run the tests.

Comment (carusogabriel) [reply test/classes/phing/components/TargetTest.php]: @siad007 This test started to fail due to `assertContains` doesn't work with the `$offset` parameter.

**Pull request merged**